extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -156,14 +156,15 @@ export function convertSlashesForSqlProj(filePath: string): string {
 /**
  * Read SQLCMD variables from xmlDoc and return them
  * @param xmlDoc xml doc to read SQLCMD variables from. Format must be the same that sqlproj and publish profiles use
+ * @param publishProfile true if reading from publish profile
  */
-export function readSqlCmdVariables(xmlDoc: any): Record<string, string> {
+export function readSqlCmdVariables(xmlDoc: any, publishProfile: boolean): Record<string, string> {
```

Comment:
did this in a separate PR since there were a lot of changes needed after adding the Document type https://github.com/microsoft/azuredatastudio/pull/17539